README.md

```diff
@@ -1,2 +1,36 @@
 # portfolio
+
 advanced css: portfolio
+
+## Creating a professional portfolio
+
+This task had involved creating my own professional portfolio. I wrote the HTML and CSS code required for a responsive layout that adapts to my viewports .
+
```

Comment:
I may of mentioned this before, but consider adding more detail to your ReadMe's. This is okay, but it would be nice to see more detail about how you approached this.